cardiac.py

```diff
@@ -67,9 +67,9 @@ def run(self, gameDisplay, info):
         sound = True
         try:
             pygame.mixer.init()
-        except Exception, err:
+        except Exception as err:
             sound = False
-            print 'error with sound', err
+            print('error with sound', err)
 
```

Comment:
This should either be removed or be moved to main.game.make (if errors are there while you removed it) . @quozl meant that `pygame.mixer.init()` should not be initialized many times. Also sugargame initializes the pygame modules within itself. so it is not explicit need to initialize the mixer (pygame's audio management system). It can be better be removed. (see my ports on `IqActivity` and `falabracman-activity` on how to do this. Falabracman has a mixer example too.